src/store/index.js

```diff
@@ -569,6 +559,31 @@ function updateWidgetData(data) {
   state.appearanceGroupOverlay.data = data
 }
 
+function findDependencies(configurations, currentField) {
+  const result = []
+
+  function recursiveFind(cField) {
+    configurations.forEach((config) => {
+      config.variables.forEach((variable) => {
+        variable.fields.forEach((field) => {
+          const inheritingVariable = isInheriting(field.default);
+
+          if (!inheritingVariable || inheritingVariable !== cField.name) {
+            return
+          }
+
+          result.push(field)
+          recursiveFind(field)
+          return
```

Comment:
what's this return for

Reply:
Now that you point it out, I realise it doesn't do anything.